river/tree/base_hoeffding_tree.py

```diff
@@ -23,11 +23,11 @@
     GRAPHVIZ_INSTALLED = False
 
 
-class BaseHoeffdingTree(ABC):
+class HoeffdingTree(ABC):
```

Comment:
I would rather keep the name as `BaseHoeffdinTree` as we do in other places (such as for ensembles). As previously mentioned, this class should not be public.

Reply:
Regardless of making this class public or not, I still do prefer shorter names hahaha

I think we need to define which pattern we are going to use across the codebase. For instance, we have:

- `base.Classifier` (and its friends)
- `metrics.Metric`

but we also have:

- `ensemble.BaseForest`
- `base.WrapperMixin`

I don't think this is a pressing matter whatsoever, but it would be nice to use the same pattern everywhere. Personally, I prefer using shorter names as much as possible: Classifier, Metric, Forest, HoeffdingTree, etc.

What do you think, @jacobmontiel and @MaxHalford ?

Reply:
Actually my opinion is that we should not add the `Base` prefix. In any case these classes are not to be user-facing.